src/Sulu/Component/Content/Compat/LocalizationFinder.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Compat;
+
+use Sulu\Component\Localization\Localization;
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+
+/**
+ * Find best localization.
+ */
+class LocalizationFinder
```

Comment:
Didn't we have an interface for this class once? Maybe we need to implement other strategies again in the future (I thought there were already different implementations)